packages/framework-provider-local/src/library/events-adapter.ts

```diff
@@ -29,6 +29,7 @@ export async function readEntityEventsSince(
   const query: object = {
     entityID: entityID,
     entityTypeName: entityTypeName,
+    kind: 'event',
```

Comment:
This fixes the bug "no reducer registered for event" in Ask.me demo. This method should return only events and not snapshots as it was happening before the fix. Returning snapshots here breaks the code and that was the bug. Now only events are returned in the query